newIDE/app/public/External/Piskel/style.css

```diff
@@ -0,0 +1,59 @@
+/**
+ * RESET
+ */
+html, body {
+  font-family: Helvetica;
+}
+
+/* ul, li {
```

Comment:
@blurymind Is this file part of Piskel? If not can I remove the unnecessary comments? :)

Reply:
I think it was a part of the original example file for embedding piskel in a web page. Since I made an entirely new file to do that, and with your refactoring - I think this can be removed :)